build_scripts/build_linux_deb-2-installer.sh

```diff
@@ -62,49 +60,64 @@ dpkg-deb --build --root-owner-group "dist/$CLI_DEB_BASE"
 cp -r dist/daemon ../chia-blockchain-gui/packages/gui
 
 # Change to the gui package
-cd ../chia-blockchain-gui/packages/gui || exit
+cd ../chia-blockchain-gui/packages/gui
```

Comment:
Why are you removing the `|| exit`? This ensures that if the directory doesn't exist that we expect to exist, the script exits (I think usually i see `exit 1`, but either way, if we can't change to an expected directory, we should not continue)

Reply:
I thought `|| exit` suppress error if directory does not exist.
If directory doesn't exist, then `cd` should fail and CI action should also fails.
I'll check more.

Reply:
`|| true` will suppress errors. Exit ensures the script exits, though it should probably be `exit 1` if we want the actual job to fail as well, not just exit

Reply:
OK. I've updated the code using `|| exit 1`.